### ISSUE TITLE
Boost Software License?

### ISSUE THREAD

**Issue by vinniefalco** [pokowaka/jwt-cpp#43]
Would it be possible to also release this under the Boost Software License? Some employers only allow the BSL (Boost license).

**Comment by pokowaka** [comment]
Looks like those licenses are very similar. Would a dual license help? I don't want to surprise some other users by changing the license.

Slide 11 & 12 of https://events.static.linuxfound.org/sites/events/files/Introduction%20to%20SPDX-without%20graphics.pdf go into some details.

**Comment by vinniefalco** [comment]
> Would a dual license help?

Yes, it would help very much. The difference between MIT and BSL is that the MIT license requires the copyright notice on binaries. BSL is the most permissive / free software license that exists. Thank you for considering it!

**Comment by vinniefalco** [comment]
I'm working on this by the way, which is pretty much finished except for the docs, which are 80% finished:
https://vinniefalco.github.io/doc/json/
https://github.com/vinniefalco/json/

**Comment by auronpriestley** [comment]
We would like to use jwt-cpp with vinnie's proposed json lib. If you would consider the dual license that would be great! Since BSL gives users more freedom, it could work as the single license for your library.
